play/settings/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for play project.
+
+Generated by 'django-admin startproject' using Django 2.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'k_yymwtq2*b3q2_jkn2d7!hiw!*hu4f^wl(v33-#*o0k-$@-g6'
```

Comment:
What's the plan here?

Reply:
the plan is to pull this out of an environment variable eventually. This was the default generated string. I was going to make another pull request with basic changes to the generated code. followed by models/migrations.